tests/docs/skip_some.rst

```diff
@@ -80,3 +80,73 @@ Code in doctest should run only if version condition is satisfied:
 .. doctest-requires:: pytest>=1.0 pytest>=2.0
 
     >>> import pytest
+
+
+Remote data block code sandwiched in block codes
+===============================================
+
+This code block should work just fine::
+
+    >>> 1 + 1
+    2
+
+This should be skipped when remote data is not requested otherwise the test should fail::
+
+.. doctest-remote-data::
+    >>> 1 + 3
+    2
+
+This code block should work just fine::
+
+    >>> 1 + 1
```

Comment:
How would we know if this is actually silently skipped too, say, because it thinks this code is still somehow part of remote date block above? That is, how are we sure the remote data block really ended before this line?

Reply:
So a blank line within a doctest separates one chunk of code from another in the case of ..doctes-remote-data:: i.e the directive considers the chunk of code after a blank line as a separate chunk of code. That way we know that it does not control the chunk of code.